### ISSUE TITLE
Decide communication path for SDR, webUSB, WASM, Rust

### ISSUE THREAD

**Issue by potto216** [user-name-is-taken/jsdr#2]
Should we first go SDR->webUSB (raw data)->Javascript(shim)->WASM(raw data)->Rust(processing)->JavaScript (graphics) or can we skip the JavaScript shim and have a flow of SDR->webUSB (raw data)->WASM(raw data)->Rust(processing)->JavaScript (graphics). 

Both of these projects seem to use a JavaScript shim
https://github.com/cho45/hackrf-sweep-webusb
https://github.com/marcnewlin/webusb-libusb-shim

**Comment by user-name-is-taken** [comment]
If we can read, process, and graph on different threads it probably doesn't matter if we use a Javascript shim. Obviously that opens up it's own can of worms

**Comment by potto216** [comment]
Good point go the shim route. It looks like  https://github.com/cho45/hackrf-sweep-webusb method would be easiest. What do you think?
